ghcid-ng/src/event_filter.rs

```diff
@@ -139,13 +137,13 @@ pub fn file_events_from_action(action: &Action) -> miette::Result<Vec<FileEvent>
                     Tag::FileEventKind(FileEventKind::Modify(ModifyKind::Name(_))) => {
                         renamed = true;
                     }
-                    Tag::FileEventKind(FileEventKind::Modify(ModifyKind::Data(_))) => {
+                    Tag::FileEventKind(FileEventKind::Modify(_)) => {
```

Comment:
I found a few situations where these were set to different values, resulting in file events going undetected.